include/fps_limit.h

```diff
@@ -0,0 +1,18 @@
+#ifndef FPS_LIMIT_H
+#define FPS_LIMIT_H
+
+#include <stdint.h>
+#include <time.h>
+
+struct fps_limit_state {
+	struct timespec last_time;
```

Comment:
Maybe a better name would be `frame_start_time`?

Reply:
Good idea, updated.

Reply:
FYI https://github.com/emersion/xdg-desktop-portal-wlr/pull/74/commits/af7f5b29befafa0b6664d6814b8f8a4445019139